pkg/templates/externalccm/digitalocean.go

```diff
@@ -174,6 +174,8 @@ func doDeployment() *appsv1.Deployment {
 					},
 				},
 				Spec: corev1.PodSpec{
+					DNSPolicy:          corev1.DNSDefault,
+					HostNetwork:        true,
```

Comment:
does it really need hostnetwork?

Reply:
I'm not sure at all. I added it since it's present in the manifest https://github.com/digitalocean/digitalocean-cloud-controller-manager/blob/16c082483c473e72bee2ec188894bbc8d8d3f753/releases/v0.1.20.yml#L17-L18.

Reply:
we overcome problem of circular dependency (CCM vs CNI) with:

```
      - effect: NoSchedule
        key: node.cloudprovider.kubernetes.io/uninitialized
        value: "true"
```

Reply:
Fixed.